e2e/app/element/select.ts

```diff
@@ -23,18 +23,26 @@ export default class Select extends BaseElement {
     super(page, xpath);
   }
 
-  async selectOption(optionValue: string): Promise<string> {
+  async selectOption(value: string): Promise<string> {
```

Comment:
The original way of finding select option is relying on the `value`. however, values are different for the Dataset on test and staging. see screenshots below. I had to change it to use displayed texts instead of value.

test env.
<img width="384" alt="Screen Shot 2020-08-18 at 10 30 24 PM" src="https://user-images.githubusercontent.com/35533885/90809562-ab773980-e2ef-11ea-9f19-33fb3a7f754d.png">

staging env.
<img width="403" alt="Screen Shot 2020-08-20 at 2 15 17 PM" src="https://user-images.githubusercontent.com/35533885/90809582-b3cf7480-e2ef-11ea-899d-c575769d7722.png">